### PR TITLE
add kotlin injections for android

### PR DESCRIPTION
**Pull request by ElenaDiachenko**
## Description

- Small description

## Related issues

- GH issues

## Npm releases

n/a

### CODE REVIEW

File: packages/core/src/schema/plugins/fragments/platformBase.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { z } from 'zod';
 // DEPRECATED?
 const Enabled = z.boolean().default(true).describe('Marks plugin platform enabled or disabled');
 const Disabled = z.boolean().default(false).describe('Marks plugin platform disabled');
+const ForceLinking = z.boolean().default(false).describe('Marks plugin as not able to be autolinked');
```
Comment (mihaiblaga89) [inline packages/core/src/schema/plugins/fragments/platformBase.ts:6]: please add a more detailed description, something like "Packages that cannot be autolinked yet can still be added to MainApplication PackageList dynamically by setting this to true"

File: packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MainActivity.kt [hunk 2]
```diff
@@ -1,10 +1,16 @@
 package {{APPLICATION_ID}}
 
+import android.os.Bundle
```
Comment (mihaiblaga89) [inline packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MainActivity.kt:3]: do we need these for every app?

File: packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MyAppPackage.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package renative.harness.debug
```
Comment (mihaiblaga89) [inline packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MyAppPackage.kt:1]: why is this in the android platform template? It's called MyPackage and references harness

**Review by mihaiblaga89 — CHANGES_REQUESTED** [review]

File: packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MyAppPackage.kt (hunk 3 above)
Comment (ElenaDiachenko) [reply packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MyAppPackage.kt:1]: templatePath is now hardcoded. I can bring back a structure based on appId. Are you sure we need such a complex folder structure?

File: packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MainActivity.kt (hunk 2 above)
Comment (ElenaDiachenko) [reply packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MainActivity.kt:3]: We define the onCreate method in MainActivity.kt and pass the Bundle. I can make this whole block dynamic. In this case, we don't need this import for all applications.

**Review by mihaiblaga89 — APPROVED** [review]

**Pull request merged**